.travis.yml

```diff
@@ -1,86 +0,0 @@
-language: cpp
-
-services:
-- docker
-
-stages:
-  - canary
-  - test
-  - coverage
-  - deploy
-
-env:
-    - DISTRO=ubuntu BUILD_UG=ON XTC=ON
-    - DISTRO=ubuntu_rolling XTC=ON
-    - DISTRO=ubuntu_mpich XTC=ON
-    - DISTRO=fedora XTC=ON
-    - DISTRO=fedora_mpich XTC=ON
-    - DISTRO=opensuse XTC=ON
-
-script:
-  - if [[ ${TRAVIS_BRANCH} = master ]]; then export DOCKER_TAG=latest; else export DOCKER_TAG="${TRAVIS_BRANCH}"; fi
-  - deployment/build.sh
-
-jobs:
-    exclude:
-        - compiler: clang
-          env: DISTRO=ubuntu_rolling XTC=OFF
-        - compiler: clang
-          env: DISTRO=fedora_mpich XTC=ON
-    include:
-        - stage: canary
-          env: DISTRO=ubuntu
-        - stage: coverage
-          env: DISTRO=ubuntu COVERAGE=ON
-        - stage: deploy
-          if: type != pull_request
-          env: DISTRO=ubuntu BUILD_UG=ON XTC=ON
-
-after_success:
-  - if [[ "${TRAVIS_BUILD_STAGE_NAME}" != [dD]eploy ]]; then travis_terminate 0; fi
-  - shopt -s extglob && [[ ${TRAVIS_BRANCH} = @(master|v[12].*) ]] && DEPLOY=yes
-  - if [[ ${DOCKER_USERNAME} && ${DOCKER_PASSWORD} && ${TRAVIS_PULL_REQUEST} == false && ${DEPLOY} ]]; then
```

Comment:
We need to replace the docker deploy as well.

Reply:
Is the docker deploy run for EVERY master commit or only for releases?

Reply:
yes